### PR TITLE
feat(general): add flag for summary position

### PR DESCRIPTION
**Pull request by marynaKK**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

[//]: # "
    # PR Title
    Be aware that we use the title to create changelog automatically and therefore only allow specific prefixes
    - break:    to indicate a breaking change, this supersedes any of the types
    - feat:     to indicate new features or checks
    - fix:      to indicate a bugfix or handling of edge cases of existing checks
    - docs:     to indicate an update to our documentation
    - chore:    to indicate adjustments to workflow files or dependency updates
    - platform: to indicate a change needed for the platform
    Additionally a scope is needs to be added to the prefix, which indicates the targeted framework, in doubt choose 'general'.
    ex.
    feat(terraform): add CKV_AWS_123 to ensure that VPC Endpoint Service is configured for Manual Acceptance
"

## Description

Fixes https://github.com/bridgecrewio/checkov/issues/3222

### Description
Added a flag to enable changing the summary position within the report. Default behavior didn't change - summary at the top. adding the flag --summary-position-bottom will move it to the bottom.

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my feature, policy, or fix is effective and works
- [x] New and existing tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by arielkru** [comment]
Very nice @marynaKK !
I see James suggested the default position to be at the bottom and it makes sense to me, shouldn't we implement it that way?
https://github.com/bridgecrewio/checkov/issues/3222#issuecomment-1184213825

**Comment by marynaKK** [comment]
@arielkru You make sense :) I thought to preserve the current behavior, the flag default is top, and if it is used the summary moves to the bottom.

**Review by nimrodkor — COMMENTED** [review]
Is there a test for this @marynaKK ? If not - can we write one?

**Comment by gruebel** [comment]
if we change the default behaviour, then we should mark this PR as a breaking change -> `break(general): ...`

File: checkov/main.py [hunk 1]
```diff
@@ -574,6 +574,8 @@ def add_parser_args(parser: ArgumentParser) -> None:
                env_var='CKV_SECRETS_SCAN_BLACK_LIST',
                action='append',
                help='black file list to filter out from the secret scanner')
+    parser.add('--summary-position-bottom', action='store_true',
```
Comment (gruebel) [inline checkov/main.py:577]:
```suggestion
    parser.add('--summary-position', choices=["top", "bottom"],
```
we could change it to allow two different choices top or bottom and then set the default to top or bottom. of course define them as constants and then they can be evaluated before printing to console.

**Review by nimrodkor — APPROVED** [review]
No tests in that area then... Only update the docs:
https://github.com/bridgecrewio/checkov/blob/96962a3c3b2301b4c1567ded3ffa7765767f26a7/docs/2.Basics/CLI%20Command%20Reference.md#L8
@gruebel This is **not** a breaking change, only if you set the arg do you get a different behavior

**Review by gruebel — COMMENTED** [review]
nice, great job on the first PR 💪

**Comment by gruebel** [comment]
@nimrodkor that's why I wrote, if we change the default behaviour 🙂 if we don't, we don't need 😄

**Comment by marynaKK** [comment]
@gruebel Your suggestion is more clear - I'll keep it in mind to keep things simpler :)
changed it.

**Review by gruebel — APPROVED** [review]
great work ☕ 🍰

**Pull request merged**